### ISSUE TITLE
Can not store the received uplinks to MongoDB.

### ISSUE THREAD

**Issue by kelleann** [gotthardp/lorawan-server#790]
Why cannot receive the received uplinks to MongoDB? Is it because of the wrong configuration? My configuration is as follows.
![1](https://user-images.githubusercontent.com/29144847/130710837-48e5def8-e174-4f49-be60-0655ee399bd7.jpg)
![2](https://user-images.githubusercontent.com/29144847/130710847-dd9002ac-fdce-4251-bc2b-5b685782d0f1.jpg)

**Comment by altishchenko** [comment]
Usually the port is 27017

**Comment by kelleann** [comment]
Yeah. You are right. But I actually configured it to 28017. Are there any other issues?
